.github/styles/Vocab/Base/accept.txt

```diff
@@ -106,4 +106,5 @@ Yevgeny
 Airbyte
 Callout
 Fargate
-gists
\ No newline at end of file
+gists
+unencrypted
```

Comment:
Referenced in https://github.com/cloudquery/cloudquery/tree/e4079914772d8191639b9935aa5970b8e27b082f/plugins/source/aws#find-all-unencrypted-rds-instances